<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="medalTab">
     <attribute name="title">
      <string>Medals</string>
     </attribute>
     <widget class="QCheckBox" name="goldCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gold Medals</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="silverCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Silver Medals</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="bronzeCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bronze Medals</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="timeTab">
     <attribute name="title">
      <string>Time Frame</string>
     </attribute>
     <widget class="QSpinBox" name="minTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1986</number>
      </property>
      <property name="maximum">
       <number>2016</number>
      </property>
     </widget>
     <widget class="QLabel" name="minTLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> &lt; Time Frame</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxTLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt; Time Frame</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="maxTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1986</number>
      </property>
      <property name="maximum">
       <number>2016</number>
      </property>
     </widget>
     <widget class="Line" name="separator">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>41</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="activateUnder">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Activate</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="activateUpper">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Activate</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="activateBoth">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Activate Both</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="activateTime">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Activate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="teamTab">
     <attribute name="title">
      <string>Team</string>
     </attribute>
     <widget class="QCheckBox" name="countryActivate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Activated</string>
      </property>
     </widget>
     <widget class="QComboBox" name="countryComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="sqlRequestBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>[SQL REQUEST HERE]</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
